extensions/resource-deployment/src/services/tools/kubeCtlTool.ts

```diff
@@ -42,8 +49,9 @@ export class KubeCtlTool extends ToolBase {
 	protected getVersionFromOutput(output: string): SemVer | undefined {
```

Comment:
Tests!

Reply:
I'll increase the test coverage for this extension next week when I am the DRI.